core/src/sigverify.rs

```diff
@@ -102,7 +102,7 @@ fn verify_packet(packet: &Packet) -> u8 {
 }
 
 fn verify_packet_disabled(_packet: &Packet) -> u8 {
-    warn!("signature verification is disabled");
+    debug!("signature verification is disabled");
```

Comment:
You might just remove this altogether, there is already a debug! in ed25519_verify_disabled. Maybe a warn! at the start of fullnode when sigverify_disabled=true.